src/software/ai/hl/stp/play/play.cpp

```diff
@@ -2,6 +2,8 @@
 
 #include <munkres/munkres.h>
 
+#include <numeric>
```

Comment:
nit: what is this `<numeric>` include for?

Reply:
Good call. it was extra and it's been removed